### ISSUE TITLE
RFC-3339 Allow The Option of 'T' or space for Date Time.

### ISSUE THREAD

**Issue by HunterEl** [justinrainbow/json-schema#414]
Hi There!

I have been using your library for about a year now and it has been fantastic. Thank you. I have been working with a team that would like to use a space instead of a `T` for the Date Time format. If you look at the [RFC-3339](https://tools.ietf.org/html/rfc3339) section `5.6` and the `NOTE` at the bottom of the page:

```  
NOTE: ISO 8601 defines date and time separated by "T".
Applications using this syntax may choose, for the sake of
readability, to specify a full-date and full-time separated by
(say) a space character.
```

The file you have for parsing and validation date time for JSON Schema requires a `T` and does not allow for the option a space. I will submit a PR under this issue number and you can let me know if that works for the direction of your project. It is a minor change to the file `src/JsonSchema/Rfc3339.php` where the regular expression needs to be modified and the date format needs to be altered. 

cheers!

**Comment by erayd** [comment]
Fine with me - thanks for the PR :-). I'll take a look at that now.

**Comment by bighappyface** [comment]
Closed with #415
